hms-app/src/pages/IdeaPortal.tsx

```diff
@@ -161,12 +161,10 @@ function IdeaPortal() {
     return item.title?.toLowerCase().includes(searchTerm.toLowerCase())
   })
 
-  const data = hackathonList.hackathons.map(
-    (hackathon, index) => hackathon.title
-  )
+  const data = hackathonList.users.map((hackathon, index) => hackathon.title)
 
   const selectChange = (value: string) => {
-    const getHackathon = hackathonList.hackathons.filter((hackathon) => {
+    const getHackathon = hackathonList.users.filter((hackathon) => {
```

Comment:
ok so I don't know exactly how to do it for these components, but normally in a dropdown list you can give the list an object and then tell it how to display the object.  Doing this means that when you select an item in the dropdown you get the object passed into the callback instead of just a string.  Then you don't need to search for the object like you are doing here